.travis.yml

```
@@ -10,11 +10,8 @@ services:
 - docker
 
 before_install:
-- curl -L https://github.com/Masterminds/glide/releases/download/0.10.2/glide-0.10.2-linux-amd64.zip
-  > glide.zip
-- unzip glide.zip
-- export PATH="$PATH:$PWD/linux-amd64"
 - go get -u github.com/golang/lint/golint
+- go get -u github.com/golang/dep/cmd/dep
```

```suggestion
DEP_RELEASE_TAG=0.5.0 curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

(That's dep's documented installation method - the script supports cross-platform)

Possible to pin to a particular version? I think dep has had breaking changes before.

---

Yes, very good hint!